### PR TITLE
Add log when server start and abnormally stop(series/2.x)

### LINKED ISSUE #19: log when Shard Manager is fully started
When start shard manager, we need a log that it is fully initialized.
Now, shard manager logs `Shard Manager loaded` when it makes `ShardManager.live`
But, after making `ShardManager.live`,  it makes api through `Server.run`.
So I'm confused if it's fully initialized.

### PR DESCRIPTION
**Pull request by wonryool-kim**
#19

### CODE REVIEW

**Comment by ghostdogpr** [comment]
Thanks! I'll review when I get back to Korea. Can you also do a PR for the 1.x branch (ZIO 1 version)?

File: manager/src/main/scala/com/devsisters/shardcake/Server.scala [hunk 1]
```diff
@@ -16,14 +17,16 @@ object Server {
...
-                         } @@ cors()
+      server       = ZServer.port(config.apiPort) ++ ZServer.app(Http.collectHttp[Request] {
+                       case _ -> !! / "health"          => Http.succeed(Response.ok)
+                       case _ -> !! / "api" / "graphql" => ZHttpAdapter.makeHttpService(interpreter)
+                       case _ -> !! / "ws" / "graphql"  => ZHttpAdapter.makeWebSocketService(interpreter)
+                     } @@ cors())
+      nothing     <- ZIO
+                       .scoped(
+                         server.make
+                           .flatMap(start => ZIO.logInfo(s"Shard cake server started on port ${start.port}.") *> ZIO.never)
```
Comment (ghostdogpr) [inline manager/src/main/scala/com/devsisters/shardcake/Server.scala]:
```suggestion
                           .flatMap(start => ZIO.logInfo(s"Shard Manager server started on port ${start.port}.") *> ZIO.never)
```

File: examples/src/main/scala/example/complex/ShardManagerApp.scala [hunk 2]
```diff
@@ -5,15 +5,20 @@ import com.devsisters.shardcake.interfaces._
...
+        ZLayer.succeed(ManagerConfig.default),
+        ZLayer.succeed(GrpcConfig.default),
+        ZLayer.succeed(RedisConfig.default),
+        redis,
+        StorageRedis.live, // store data in Redis
+        PodsHealth.local,  // just ping a pod to see if it's alive
+        GrpcPods.live,     // use gRPC protocol
+        ShardManager.live  // Shard Manager logic
+      )
+      .catchAllCause(cause =>
```
Comment (ghostdogpr) [inline examples/src/main/scala/example/complex/ShardManagerApp.scala]: How about we just keep the example simple? We can add that in our own code.

Comment (wonryool-kim) [reply examples/src/main/scala/example/complex/ShardManagerApp.scala]: [33eb5a0](https://github.com/devsisters/shardcake/pull/20/commits/33eb5a098994cdcd9408402ae2cdfcceea827687)

**Comment by ghostdogpr** [comment]
Ah my suggested code required fmt

**Review by ghostdogpr — APPROVED** [review]

**Pull request merged**